controllers/userC.go

```diff
@@ -31,9 +35,17 @@ type UserController struct {
 
 var User UserController
 
-func (c *UserController) Index(w http.ResponseWriter, r *http.Request) {
-	user := []string{"first name", "last name"}
-	if err := views.User.Index.Render(w, user); err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+func (c *UserController) Index(db *sqlx.DB) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+
+		//needs to be a slice of User from the database
+		//pagination needs to be built in from the start. get from query param if
+		//there and store in cookie. else, get from cookie
+		users, _ := models.GetAllUsers(db, 10, 0)
```

Comment:
The error from `GetAllUsers()` should be handled. You could probably just return a 500 error like: `return w.WriteHeader(http.StatusInternalServerError)` and probably would be a good idea to log the error as well.

Reply:
Yes, this is just kind of a junk placeholder demonstrating how the model and controller interact.  The function in the model doesn't even utilize the pagination options.

Reply:
Figured that might be the case. Disregard my comment then. lol